### ISSUE TITLE
Bot filter?

### ISSUE THREAD

**Issue by symac** [Daniel-KM/Omeka-plugin-Stats#3]
That might have been already thought but that might be a good idea to have an option to filter visits by bots. I have just installed the plugin and in the top 10 browsers, I have got : 

- Mozilla/5.0 (compatible; AhrefsBot/5.2; +http://ahrefs.com/robot/)
- Mozilla/5.0 (compatible; bingbot/2.0; +http://www.bing.com/bingbot.htm)
- Mozilla/5.0 (compatible; Qwantify/2.4w; +https://www.qwant.com/)/2.4w
- Mozilla/5.0 (compatible; Googlebot/2.1; +http://www.google.com/bot.html)
- Mozilla/5.0 (compatible; Yahoo! Slurp; http://help.yahoo.com/help/us/ysearch/slurp)
- Mozilla/5.0 (compatible; DotBot/1.1; http://www.opensiteexplorer.org/dotbot, help@moz.com)
- Mobile/11A465 Safari/9537.53 (compatible; bingbot/2.0; +http://www.bing.com/bingbot.htm)

Being able to filter this would provide more accurate stats. Even if there is no way to remove everything, I imagine there might be some list of bots names available that could be integrated to filter stats).

**Comment by Daniel-KM** [comment]
Yes. In fact, the ideal would be to integrate [piwik](https://piwik.org/) as a plugin in Omeka (or Omeka S) so we can have all the stats for the site and adapted stats for Omeka collections and items from the admin board.
